README.md

```diff
@@ -4,16 +4,16 @@
 
 * [Summary](#summary)
 * [Contributors](#contributors)
-* [MVP Objectives](#mvp-objectives)
-* [Functionality](#functionality)
 * [Architecture](#architecture)
-* [Past Issues](#past-issues)
-* [Known Bugs](#known-bugs)
+* [Installation](#installation)
+* [Functionality](#functionality)
+* [Known Problems](#known-problems)
+* [Contributing](#contributing)
 * [References](#references)
 
 ## Summary
 
-Our project consists of two main parts. The first part of our project is data visualization and teamwork research. We are using the CS 121 teamwork survey as our basis. From this, we will develop a web interface that shows data from this survey. We will research the best, most helpful graphs to show. To enable us to do this, we will need a database, a method for the user to upload to the database, and a way to construct graphs from this data. The second part of our project consists of researching team dynamics. We are going to research many different teamwork strategies to fine tune the CS 121 survey, as well as estimate what consists of a good team.
+This project is a teamwork analysis web app. It is designed to display results from the Parker Team Player Survey (read more about that [here](https://www.creativeorgdesign.com/tests_page.php?id=185)), which scores people as Contributors, Collaborators, Challengers, and Communicators. One can import the results from [this](https://docs.google.com/forms/d/1px6yEtC9zEG469OELr_1apL2iTUqwwDRYkoirncgVBI/edit) survey into the website. Once they have data in the database, the user can view an overall summary. This displays the primary, secondary, tertiary, and quaternary types for each student in the database. The user can also select an arbitrary number of students and view their scores for each of the four cateogries. 
```

Comment:
The google doc link will currently only work for us because link sharing on the google form is off. We should probably turn it on to preview only if we are going to put the link on this page. Also, I think that the link to the survey would make more sense if it was placed after the comment about the 4 different styles.